examples/README.md

```diff
@@ -10,6 +10,7 @@ Below is a brief description of each of these:
 | `maturin-starter` | A template project which is configured to use [`maturin`](https://github.com/PyO3/maturin) for development. |
 | `setuptools-rust-starter` | A template project which is configured to use [`setuptools_rust`](https://github.com/PyO3/setuptools-rust/) for development. |
 | `word-count` | A quick performance comparison between word counter implementations written in each of Rust and Python. |
+| `plugin` | Illustrates how to make Python plugins work within a Rust application. Allows to test plugin API separately from the main application. |
```

Comment:
This description is too long. Also, can this be useful for more things than just testing? It would be nice to use that as an example instead.

```suggestion
| `plugin` | Shows how to call a python extension module from Rust. |
```

Alternatively, you could focus on testing instead:

```suggestion
| `plugin` | Testing your python extension module from Rust. |
```

Reply:
I think the point was missed. The example is about making plugins in python and PyO3 for Rust application. It is not just about how  to test them (but also about that as if you just pile everything into one crate it becomes a nightmare to test). Imagine if you had to start an entire video editing software just to test a one line change in your python API. That would be a total disaster.